plugins/destination/duckdb/client/write.go

```diff
@@ -139,14 +186,16 @@ func (c *Client) WriteTableBatch(ctx context.Context, name string, msgs message.
 	c.logger.Debug().Str("table", table.Name).Str("duration", time.Since(writeStart).String()).Msg("write tmp file")
 	defer os.Remove(tmpFile)
 
-	if len(table.PrimaryKeys()) == 0 {
-		copyStart := time.Now()
-		defer func() {
-			c.logger.Debug().Str("table", table.Name).Str("duration", time.Since(copyStart).String()).Msg("copy file to table")
-		}()
-		return c.copyFromFile(ctx, name, tmpFile, table)
-	}
-
+	/*
+		if len(table.PrimaryKeys()) == 0 {
```

Comment:
remove?

Reply:
I uncomment it from time to time to test direct parquet copys.